### PR TITLE
Extracted euronews streams

### PR DESCRIPTION
**Pull request by pimboli**

### CODE REVIEW

File: streams/euronews.m3u [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+#EXTM3U
+#EXTINF:-1 tvg-id="EuronewsEnglish.fr" status="online",Euronwews French
```
Comment (BountyChocolate123456) [inline streams/euronews.m3u:3]:
```suggestion
#EXTINF:-1 tvg-id="EuronewsFrench.fr" status="online",Euronews French
```

File: streams/euronews.m3u [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+https://euronews.alteox.app/hls/fr_stream.m3u8
+#EXTINF:-1 tvg-id="EuronewsItalian.fr" status="online",Euronews Italian
+https://euronews.alteox.app/hls/it_stream.m3u8
+#EXTINF:-1 tvg-id="EuronewsPortuguese.fr" status="online",Euronews Portugese
+https://euronews.alteox.app/hls/pt_stream.m3u8
+#EXTINF:-1 tvg-id="EuronewsSpanish.fr" status="online",Euronews Spanish
+https://euronews.alteox.app/hls/es_stream.m3u8
+#EXTINF:-1 tvg-id=EuronewsGerman.fr" status="online",Euronews German
```
Comment (BountyChocolate123456) [inline streams/euronews.m3u:11]:
```suggestion
#EXTINF:-1 tvg-id=EuronewsEnglish.fr" status="online",Euronews English
https://euronews.alteox.app/hls/en_stream.m3u8
#EXTINF:-1 tvg-id=EuronewsGerman.fr" status="online",Euronews German
```

File: streams/euronews.m3u [hunk 3]
```diff
@@ -0,0 +1,14 @@
+
+#EXTM3U
+#EXTINF:-1 tvg-id="EuronewsFrench.fr" status="online",Euronews French
+https://euronews.alteox.app/hls/fr_stream.m3u8
```
Comment (BountyChocolate123456) [inline streams/euronews.m3u:4]:
```suggestion
https://euronews.alteox.app/hls/fr_stream.m3u8
#EXTINF:-1 tvg-id="EuronewsRussian.fr" status="online",Euronews Russian
https://euronews.alteox.app/hls/ru_stream.m3u8
```

**Comment by BountyChocolate123456** [comment]
Thanks @pimboli for your help. I fixed the Euronews French tvg-id and added Euronews English & Euronews Russian

**Review by ghost — CHANGES_REQUESTED** [review]

File: streams/euronews.m3u [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+#EXTINF:-1 tvg-id="EuronewsRussian.fr" status="online",Euronews Russian
+https://euronews.alteox.app/hls/ru_stream.m3u8
+#EXTINF:-1 tvg-id="EuronewsItalian.fr" status="online",Euronews Italian
+https://euronews.alteox.app/hls/it_stream.m3u8
+#EXTINF:-1 tvg-id="EuronewsPortuguese.fr" status="online",Euronews Portugese
+https://euronews.alteox.app/hls/pt_stream.m3u8
+#EXTINF:-1 tvg-id="EuronewsSpanish.fr" status="online",Euronews Spanish
+https://euronews.alteox.app/hls/es_stream.m3u8
+#EXTINF:-1 tvg-id=EuronewsEnglish.fr" status="online",Euronews English
```
Comment (BountyChocolate123456) [inline streams/euronews.m3u:12]:
```suggestion
#EXTINF:-1 tvg-id="EuronewsEnglish.fr" status="online",Euronews English
```

File: streams/euronews.m3u [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+https://euronews.alteox.app/hls/en_stream.m3u8
+#EXTINF:-1 tvg-id=EuronewsGerman.fr" status="online",Euronews German
```
Comment (BountyChocolate123456) [inline streams/euronews.m3u:14]:
```suggestion
#EXTINF:-1 tvg-id="EuronewsGerman.fr" status="online",Euronews German
```

**Review by ghost — DISMISSED** [review]

**Review by its0x08 — DISMISSED** [review]

**Review by what-a — DISMISSED** [review]

**Review by freearhey — CHANGES_REQUESTED** [review]
Please change the file name to `fr_euronews.m3u` to match the others.

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by what-a — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**